08-communicating-results.Rmd

```diff
@@ -297,7 +303,7 @@ anes_des_der <- anes_des %>%
 anes_des_der
 ```
 
-Now, we can begin creating our chart with {ggplot2}. First, we set up our plot with `ggplot()`. Next, we state the data points we want to show with `aes`. Finally, we specify the type of plot with `geom_*()`, in this case, `geom_bar()`.
+Now, we can begin creating our chart with {ggplot2}. First, we set up our plot with `ggplot()`. Next, we state the data points we want to show with `aes`. Finally, we specify the type of plot with `geom_*()`, in this case, `geom_bar()`. This initial plot is displayed in Figure \@ref(results-plot1).
 
 ```{r}
 #| label: results-plot1
```

Comment:
All plots in PDF have additional text under them.  I think this might be the alt text that is getting printed or errors associated with BrailleR.  Need to look into this more and figure out how to remove them from the PDF.